aws/aws-application-networking-k8s
# Add GRPCRoute e2etest cases

**What type of PR is this?**



**Which issue does this PR fix**: 

- Add GRPCRoute e2etest cases
- Make sure test framework could load GRPCRoute CRD and TargetGroupPolicy CRD
- Use gwlog in the test framework


**Testing done on this change**:


**Automation added to e2e**:

Newly added e2etest cases can pass, all e2etest cases can pass

```
------------------------------

Ran 15 of 15 Specs in 1844.288 seconds
SUCCESS! -- 15 Passed | 0 Failed | 0 Pending | 0 Skipped
```

**Will this PR introduce any new dependencies?**:


**Will this break upgrades or downgrades. Has updating a running cluster been tested?**: No

**Does this PR introduce any user-facing change?**: No


```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
